src/pagination/pagination.ts

```diff
@@ -7,41 +7,46 @@ import {getValueInRange, toInteger, toBoolean} from '../util/util';
 @Component({
   selector: 'ngb-pagination',
   changeDetection: ChangeDetectionStrategy.OnPush,
+  styles: [`
+    button.page-link {
+      outline: 0;
```

Comment:
Using buttons? Or removing outline?

What is your alternative proposal? (hint: if you don't remove outline it looks really ugly - try clicking on prev / next buttons).